### DISCUSSION TITLE
Lost Data (For Me - Orders)

### DISCUSSION THREAD

**Discussion by briantofu** [saleor/saleor#5753] [Q&A] [answered]
### Api Problem
We have encountered a problem where our order data has lost. This is on me orders query. How we could fix this issue?

Apparently we updated some modifications of the api by Saleor.
Thanks!

**Comment by maarcingebala** [comment, marked as answer]
Please provide more details or steps to reproduce the problem, it's hard to tell what could be the issue from the brief description.
